util/circular-chk.py

```diff
@@ -0,0 +1,51 @@
+import glob
+import re
+from sys import exit, argv
+from pathlib import Path
+
+strict = False
+if len(argv) > 1 and argv[1] == '--strict' :
+    strict = True
+
+regexExclude = r"keymaps\/(.*)via\/"
+reg = re.compile(regexExclude)
```

Comment:
We avoid the use of regex whenever possible.

Reply:
so how would change this? this is fairly simple expression.

Reply:
Farther down I have a suggestion that eliminates the need for the regex.